docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md

```diff
@@ -27,8 +27,13 @@ You can register a task to run any of the following application or file types: W
 ## EXAMPLES
 
 ### Example 1: Define a scheduled task and register it at a later time
-```
-PS C:\> $A = New-ScheduledTaskAction -Execute "Taskmgr.exe" PS C:\>$T = New-ScheduledTaskTrigger -AtLogon PS C:\>$P = "Contoso\Administrator" PS C:\>$S = New-ScheduledTaskSettingsSet PS C:\>$D = New-ScheduledTask -Action $A -Principal $P -Trigger $T -Settings $S PS C:\>Register-ScheduledTask T1 -InputObject $D
+```powershell
+PS C:\> $A = New-ScheduledTaskAction -Execute "Taskmgr.exe"
+PS C:\> $T = New-ScheduledTaskTrigger -AtLogon
+PS C:\> $P = "Contoso\Administrator"
+PS C:\> $S = New-ScheduledTaskSettingsSet
+PS C:\> $D = New-ScheduledTask -Action $A -Principal $P -Trigger $T -Settings $S
+PS C:\> Register-ScheduledTask T1 -InputObject $D
```

Comment:
Instead of using letters, can you use full names for the variables, similar to what you've done for example 2?

Reply:
Thats the original example. I just added powershell formatting from the other version. I can update it as well.

Reply:
No problem, if you could use full names instead of letters that's helpful for examples :-)